scripts/statistics.js

```diff
@@ -12,7 +12,7 @@ const bcd = require('..');
 
 const browsers = ['chrome', 'chrome_android', 'edge', 'firefox', 'ie', 'safari', 'safari_ios', 'webview_android'];
 /** @type {{total: VersionStats; [browser: string]: VersionStats}} */
-let stats = { total: { all: 0, true: 0, null: 0, range: 0, real: 0 } };
+const stats = { total: { all: 0, true: 0, null: 0, range: 0, real: 0 } };
```

Comment:
`const` indicates that a variable will never change, but we're definitely altering the contents in this file.  I think `let` is more appropriate here.

Reply:
We don't really have a codebase style in BCD for this sort of thing, but more of an expectational thing, at least for me.  Even if `const` may allow for the modification of objects, the keyword tells me "this is meant to be assigned once and never change".  It's mostly just a personal preference on that one -- thanks for changing it back!